packages/payments/package.json

```
@@ -1,51 +1,52 @@
 {
   "name": "@graphprotocol/payments",
   "version": "0.0.1",
-  "description": "Graph Network State Channel Payment Management",
+  "description": "State Channel Payment Manager",
   "repository": "https://github.com/statechannels/graph-payments",
-  "author": "State Channels",
+  "author": "Graph Protocol, Inc.",
```

Unintentional change:

```suggestion
  "author": "State Channels",
```